### PR TITLE
Fixed quotation marks and commented out 'SCE Team'

### PR DESCRIPTION
**Pull request by Annabel1054**
Commented out the 'SCE Team' navbar from Users' view, and refactored test-cases.

### CODE REVIEW

**Comment by evanugarte** [comment]
@Annabel1054  It looks like prettier changed the quotations and added semicolons. Do you wanna meet later to sort this out?

File: src/Components/Navbar/UserNavbar.js [hunk 1]
```diff
@@ -40,11 +40,11 @@ export default function UserNavBar (props) {
...
       ].join('')
     }
-  ]
+  ];
   const unauthedRoutes = [
-    { title: 'Events', route: '/events' },
-    { title: 'SCE Team', route: '/Team' }
-  ]
+    { title: 'Events', route: '/events' }
+    //{ title: 'SCE Team', route: '/Team' }
```
Comment (evanugarte) [inline src/Components/Navbar/UserNavbar.js]: remove this comment

File: src/Components/Navbar/UserNavbar.js [hunk 2]
```diff
@@ -40,11 +40,11 @@ export default function UserNavBar (props) {
...
+  ];
```
Comment (evanugarte) [inline src/Components/Navbar/UserNavbar.js]: no semicolons

File: src/Components/Navbar/UserNavbar.js [hunk 3]
```diff
@@ -64,7 +64,7 @@ export default function UserNavBar (props) {
                   <path fill='#757575' d={icon.vector} />
                 </svg>
               </a>
-            )
+            );
```
Comment (evanugarte) [inline src/Components/Navbar/UserNavbar.js]: go through this file and remove the semi colons

File: src/Components/Navbar/AdminNavbar.js [hunk 4]
```diff
@@ -1,36 +1,36 @@
...
   Navbar,
   NavbarBrand,
   NavLink,
   Nav,
   Collapse,
   NavbarToggler
-} from "reactstrap";
-import { membershipState } from "../../Enums";
+} from 'reactstrap';
+import { membershipState } from '../../Enums';
```
Comment (evanugarte) [inline src/Components/Navbar/AdminNavbar.js]: go through this file and remove the semi colons

File: test/frontend/AdminNavbar.test.js [hunk 5]
```diff
@@ -1,35 +1,35 @@
 /* global describe it */
-import "jsdom-global/register";
-import React from "react";
-import Enzyme, { mount } from "enzyme";
-import { expect } from "chai";
+import 'jsdom-global/register';
```
Comment (evanugarte) [inline test/frontend/AdminNavbar.test.js]: go through this file and remove the semi colons

File: test/frontend/UserNavbar.test.js [hunk 6]
```diff
@@ -32,29 +32,29 @@ describe('<UserNavbar />', () => {
...
   it('Should render four <NavLink /> tags for unauthenticated routes', () => {
     const wrapper = mount(<UserNavbar />)
-    expect(wrapper.find(NavLink)).to.have.lengthOf(3)
+    expect(wrapper.find(NavLink)).to.have.lengthOf(2)
     expect(wrapper.find(Nav).children()).to.have.lengthOf(1)
   })
   it(
     'Should render two <NavLink /> components for' +
-      " the user who isn't a member",
+    " the user who isn't a member",
```
Comment (evanugarte) [inline test/frontend/UserNavbar.test.js]: make these single quotes

Comment (Annabel1054) [reply test/frontend/UserNavbar.test.js]: We can't get rid of ```""``` because ```isn't``` has a ```'```.

Comment (evanugarte) [reply test/frontend/UserNavbar.test.js]:
```suggestion
    ' the user who isn\'t a member',
```

**Pull request merged**